x-pack/plugins/session_view/public/components/tty_player_controls/tty_player_controls_markers/index.tsx

```diff
@@ -44,10 +44,9 @@ export const TTYPlayerControlsMarkers = ({
       return [];
     }
     return processStartMarkers.map(
-      ({ event, line }) =>
+      ({ event, line, maxBytesExceeded }) =>
         ({
-          type:
-            event.process?.io?.max_bytes_per_process_exceeded === true ? 'data_limited' : 'output',
+          type: maxBytesExceeded ? 'data_limited' : 'output',
```

Comment:
💎

Reply:
I'm having trouble understanding `type` does type property mean. `type`  seems like a generic name. Is it possible to store the ternary expression in a descriptive variable name?

Reply:
good call, will add some const